templates/default/ndbd-init.sh.erb

```diff
@@ -30,6 +30,10 @@ while [ $# -gt 0 ]; do
   shift       
 done
 
+SERVICE_ARG=
+if [ -n "$SERVICE_NAME" ]; then
+    SERVICE_ARG="--service-name $SERVICE_NAME"
+fi
```

Comment:
is that code needed? doesn't seem to be used in that file

Reply:
yep, needed by the ndb-agent, the service name becomes the name of the pidfile (makes it possible in the future to have multiple ndbmtd services on the same machine as well)